x-pack/test/api_integration/deployment_agnostic/services/slo_api.ts

```diff
@@ -84,9 +103,23 @@ export function SloApiProvider({ getService }: DeploymentAgnosticFtrProviderCont
       return body;
     },
 
-    async findDefinitions(roleAuthc: RoleCredentials): Promise<FindSLODefinitionsResponse> {
+    async findDefinitions(
+      roleAuthc: RoleCredentials,
+      params?: Record<string, string>
+    ): Promise<FindSLODefinitionsResponse> {
       const { body } = await supertestWithoutAuth
         .get(`/api/observability/slos/_definitions`)
+        .query(params || {})
+        .set(roleAuthc.apiKeyHeader)
+        .set(samlAuth.getInternalRequestHeader())
+        .send();
+
+      return body;
+    },
+
+    async getSavedObject(roleAuthc: RoleCredentials, id: string): Promise<SavedObjectResponse> {
```

Comment:
Just to make it clearer what id we are talking about, SLO id vs Saved Object Id. It can be confusing.
```suggestion
    async getSavedObject(roleAuthc: RoleCredentials, sloId: string): Promise<SavedObjectResponse> {
```